spec/android_update_release_notes_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe Fastlane::Actions::AndroidUpdateReleaseNotesAction do
+  describe '#android_update_release_notes' do
+    it 'adds a new section on RELEASE-NOTES.txt' do
+      in_tmp_dir do |tmp_dir|
+        # Arrange
+        ENV['PROJECT_ROOT_FOLDER'] = tmp_dir
+        release_notes_txt = File.join(tmp_dir, 'RELEASE-NOTES.txt')
+        File.write(release_notes_txt, '')
```

Comment:
It would be more resilient to have the fixture actually contain some text before we apply the action, to test if the action actually prepends to the existing content, instead of appending, or replacing the whole existing content.

Even if this use case is already covered in depth by [the existing specs for the `Fastlane::Helper::ReleaseNotesHelper`](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/spec/release_notes_helper_spec.rb) — which test the helper itself and not the wrapping actions using it like you're doing here — it doesn't hurt to make a stronger test here too.

Reply:
Yeah, my main reasoning was that this is already covered in another test. But having an empty fixture feels a little bit weird, changed in 8f1dea21b17f8920c3eb131ff8fea4b1cb535721 and cd42ef4abcf374721c61a716422d2b242b4cd45b